tools/mason/MasonUtils.chpl

```diff
@@ -158,12 +169,13 @@ proc getSpackResult(cmd, quiet=false) : string throws {
    TODO: get to work with Spawn */
 proc runSpackCommand(command) {
 
-  var prefix = "export SPACK_ROOT=" + MASON_HOME + "/spack" +
-    " && export PATH=$SPACK_ROOT/bin:$PATH" +
-    " && source $SPACK_ROOT/share/spack/setup-env.sh && ";
+  var prefix = "export SPACK_ROOT=" + SPACK_ROOT +
```

Comment:
Looking at this now, I wonder if we can use the `env` argument for `spawnshell` rather than setting it in the command. I will test this tomorrow, unless you beat me to it.

Reply:
Yes, especially since I don't think we need PATH, so this is a single line. In fact, we probably don't even need SPACK_ROOT set explicitly, in which case it's even simpler.